source/crud/tests/unified/bulkWrite-comment.yml

```diff
@@ -48,15 +48,15 @@ tests:
             - updateOne:
                 filter: &updateOne_filter
                   _id: 2
-                update: &update { x: "updated" }
+                update: &update { $set: {x: "updated"} }
```

Comment:
@comandeo: If this was missed in https://github.com/mongodb/specifications/pull/1117 I think there may be a bug in Ruby if a replacement doc with no top-level, `$`-prefixed operators was required for updateOne's `update` parameter. See [Update vs. Replace Validation](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#update-vs-replace-validation) for more info.

I also find it a bit odd that the original test would have expected separate `update` commands for the replaceOne and updateOne operations in the bulk write, especially if updateOne essentially took a replacement document; however, I don't believe there's any spec requirement to efficiently batch write operations so that's not technically a deviation (although it'd likely cause conflicts with other spec tests if other drivers are all doing optimal batching).

Reply:
It seems that there is a bug in the unified test runner in Ruby driver. I tested my tests against Ruby drive only. Thank you @juliusgeo for fixing it!